RecipeApp/Modules/RecipeBook/ViewModel/RecipeBookViewModel.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  RecipeBookViewModel.swift
+//  RecipeApp
+//
+//  Created by Hadley Igoe on 2019-12-17.
+//  Copyright © 2019 Hadley Igoe. All rights reserved.
+//
+
+import Foundation
+
+protocol RecipeBookViewModelProtocol {
+    func viewWillAppear()
```

Comment:
Nicely done and glad you grouped the funcs & vars. Usually, vars are at the top, though. (PICKY)